### PR TITLE
Basic Cumulus Linux support

### PR DESCRIPTION
**Pull request by xbeaudouin**
Hi,

Here is a basic Cumulus Linux support. More later, but this allow Librenms to detect Cumulus Linux Switches as is.

### CODE REVIEW

**Comment by paulgear** [comment]
Seems reasonable to me; anyone available to test?

**Comment by xbeaudouin** [comment]
I have done this using my s6000 at work :D

File: includes/discovery/os/linux.inc.php [hunk 1]
```diff
@@ -64,6 +64,9 @@
             );
             register_mibs($device, $pktj_mibs, "include/discovery/os/linux.inc.php");
         }
+        else if (stristr($sysObjectId, 'cumulusMib') || strstr($sysObjectId, '.1.3.6.1.4.1.40310')) {
```
Comment (laf) [inline includes/discovery/os/linux.inc.php]: Can you change this to elseif please.

File: includes/definitions.inc.php [hunk 2]
```diff
@@ -240,6 +240,18 @@
 $config['os'][$os]['over'][2]['graph'] = 'device_mempool';
 $config['os'][$os]['over'][2]['text']  = 'Memory Usage';
 
+$os = 'cumulus';
+$config['os'][$os]['type']             = 'network';
+$config['os'][$os]['group']            = 'unix';
+$config['os'][$os]['text']             = 'Cumulus Linux';
+//$config['os'][$os]['icon']             = 'cumulus';
```
Comment (laf) [inline includes/definitions.inc.php]: May as well remove this if it's not needed

Comment (xbeaudouin) [inline includes/definitions.inc.php]: Well, I am waiting for an official green light to push the icon from Cumulus Network.
Do I remove until I get it ? Please advise.

File: includes/discovery/os/linux.inc.php (hunk 1 above)
Comment (xbeaudouin) [inline includes/discovery/os/linux.inc.php]: ok

**Comment by laf** [comment]
:+1:

**Comment by xbeaudouin** [comment]
Hi ! Is there anything missing on this ? :D 
Regards,
Xavier

**Comment by dorkmatt** [comment]
Couple questions (former @cumulusnetworks employee and active user of Cumulus Linux with @sfmix), I haven't seen the sysObjectId return back 'cumulusMib' before - the enterprise ID should be enough?

Also, I slacked on merging in a version of this from my own repo - see https://github.com/dorkmatt/librenms/commit/25acd8c8097fba2ad4c0b80459d4fe6d45fb8b88 Can we consider the close-up logo, I can hardly see the version here that is committed. Thank you.

**Comment by laf** [comment]
@dorkmatt thanks for the comments. Feel free to submit an updated PR with the logo update.

@xbeaudouin any thoughts on the sysObjectID?

**Pull request merged**